backend/app/Observers/CommentObserver.php

```diff
@@ -12,6 +12,9 @@ class CommentObserver
 
     public function created(Comment $comment)
     {
+        if (! auth()->check()) {
```

Comment:
is this not handled by our route authorization or policy?
Refer to this link: https://stackoverflow.com/a/63405046

Reply:
its for the seeder sir @chevyB